### ISSUE TITLE
Wrong png image of simple.er in the readme

### ISSUE THREAD

**Issue by kukimik** [BurntSushi/erd#97]
In README.md the image following the line

> The PDF should now contain a graph that looks like this:

is wrong (the names of the entity "Location" and its attributes are different than in the `simple.er` file that it illustrates). This is mildly confusing.

The image is hosted at http://burntsushi.net/stuff/erd-example-simple.png , so I can't replace it in a PR.

@BurntSushi Can you replace the image with the one below? (Or maybe I can replace the URL in the README.md with the following one https://user-images.githubusercontent.com/53443372/112312431-18e4b180-8ca7-11eb-85c8-28824c3b7a78.png  that points to the image attached to this issue?)

![simple](https://user-images.githubusercontent.com/53443372/112312431-18e4b180-8ca7-11eb-85c8-28824c3b7a78.png)

**Comment by BurntSushi** [comment]
Done. Would probably be good to use a different URL though. Maybe commit the image to the repo?

**Comment by kukimik** [comment]
Thanks!

> Maybe commit the image to the repo?

That was my first thought. However, I wasn't sure about this, because there must be some reason that the image is not already there. Maybe this is a historical reason that is no longer relevant.

If it is so, then I'll make a PR after #91 and #95 are merged to avoid merging the README.
